server/apps/microservices/src/processors/metadata-extraction.processor.ts

```diff
@@ -73,10 +72,48 @@ export type GeoData = {
   distance: number;
 };
 
+class WrapperLogger {
```

Comment:
We've already had a logger configured. Please don't create a wrapper.

See example
https://github.com/samip5/immich/blob/92eee11e8d7e9173518be4b52b8e4deea1ac0290/server/apps/immich/src/api-v1/asset/asset.service.ts#L70-L71

Reply:
Unfortunately, without a wrapper:

```
TS2322: Type '() => import("/home/sky/Github/immich/server/node_modules/@nestjs/common/services/logger.service").Logger' is not assignable to type '() => import("/home/sky/Github/immich/server/node_modules/batch-cluster/dist/Logger").Logger'.   Type 'Logger' is missing the following properties from type 'Logger': trace, info
```

The NextJS logger doesn't implement those, so what's the better way to handle that?

Reply:
We don't have to use the internal logger. The information from the logic flow would leave us enough info for trouble shooting. 

Sorry for the back and forth, we are both working too fast 😛

Reply:
True we don't have to, but then that would need to imported from some other package, as one cannot use NextJS one.

I thought that it would be better to use the internal one so that it's not out of place in the log output.

Which approach we then should take? Import the logger from somewhere else and not use a wrapper for it?

Reply:
Let's not use the wrapper for it.

Reply:
ALright, removed the wrapper and it uses the batch-cluster's default logger.

Reply:
Can you remove the Wrapper's code?

Reply:
I did already though? @alextran1502